.appveyor.yml

```diff
@@ -0,0 +1,29 @@
+clone_depth: 2
+clone_folder: c:\projects\sentry-php
+
+cache:
+  - composer.phar
+
+init:
+  - SET PATH=c:\php;%PATH%
+  - SET COMPOSER_NO_INTERACTION=1
+  - SET SYMFONY_DEPRECATIONS_HELPER=strict
+  - SET ANSICON=121x90 (121x90)
+
+install:
+  - mkdir c:\php && cd c:\php
+  - appveyor DownloadFile https://github.com/symfony/binary-utils/releases/download/v0.1/php-7.1.3-Win32-VC14-x86.zip
+  - 7z x php-7.1.3-Win32-VC14-x86.zip -y >nul
+  - echo memory_limit=-1 >> php.ini
+  - echo zend_extension=php_opcache.dll >> php.ini
```

Comment:
It should make the build run faster.

Reply:
Strange, it indeed works in a web request context since the compiled code can be shared among requests but we're in a CLI context, so we should not see any difference

Reply:
That's why there's `echo opcache.enable_cli=1 >> php.ini` a few lines after this one.

Reply:
It seems more complicated than just enabling this option. Quoting from [this](https://stackoverflow.com/questions/25044817/zend-opcache-opcache-enable-cli-1-or-0-what-does-it-do) Stackoverflow question:

> The opcache.enable_cli option enables PHP OPcache when running PHP scripts from the command line (using the php command). However, keep in mind that for PHP 5.x the OPcache extension works by storing cached opcodes in the memory of the current process. This is only useful when the process that's running PHP is going to be handling multiple requests that can reuse these opcodes, like in a web server or under FastCGI. For a process like the PHP CLI, which runs one "request" and exits, it just wastes memory and time.

and

> With PHP7 and file-based caching, it can now make sense to enable opcache for CLI. The best possibility would be to have a separate php.ini for CLI with the following configuration:
> 
> ```ini
> opcache.enable=1
> opcache.enable_cli=1
> opcache.file_cache="/tmp/php-file-cache"
> opcache.file_cache_only=1
> opcache.file_cache_consistency_checks=1
> ```

I don't think that enabling OPcache in a development environment should be done, less than that in the CI. This feature is meant to optimize and speed up things in production